govcd/metadata_test.go

```diff
@@ -0,0 +1,181 @@
+// +build vapp vdc metadata functional ALL
+
+/*
+ * Copyright 2018 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+
+	. "gopkg.in/check.v1"
+)
+
+func init() {
+	testingTags["metadata"] = "metadata_test.go"
+}
+
+func (vcd *TestVCD) Test_AddMetadataForVdc(check *C) {
```

Comment:
Could this test do a cleanup (probably using the helper `AddToCleanupList`)? Now if you run explicitly this one it will leave the `key` hang there forever if the vdc is not recycled on every test suite run.

<img width="808" alt="image" src="https://user-images.githubusercontent.com/15804230/60182654-61024600-982d-11e9-9a87-6539b613ed76.png">

Reply:
Added